.github/workflows/e2e-test.yaml

```diff
@@ -0,0 +1,28 @@
+name: Run CCNP E2E test
+on:
+   schedule:
+    - cron: '20 10 * * *'
+   workflow_dispatch:
+jobs:
+  E2E_test:
+    name: Publish ccnp package
+    runs-on: tdvm-ut
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

Comment:
v3 is too old for checkout action, before we use depot to bump to latest , we can manually assign the latest version , now is 4.1.1.

Reply:
Fixed